[2024-04-30T21:10:55.891+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-30T21:10:55.912+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_yellow_cab.load_data manual__2024-04-30T21:10:20.104666+00:00 [queued]>
[2024-04-30T21:10:55.920+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_yellow_cab.load_data manual__2024-04-30T21:10:20.104666+00:00 [queued]>
[2024-04-30T21:10:55.920+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-30T21:10:55.933+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_data> on 2024-04-30 21:10:20.104666+00:00
[2024-04-30T21:10:55.939+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=96) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-30T21:10:55.940+0000] {standard_task_runner.py:63} INFO - Started process 98 to run task
[2024-04-30T21:10:55.940+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_yellow_cab', 'load_data', 'manual__2024-04-30T21:10:20.104666+00:00', '--job-id', '280', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp5ie47ejq']
[2024-04-30T21:10:55.943+0000] {standard_task_runner.py:91} INFO - Job 280: Subtask load_data
[2024-04-30T21:10:55.988+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_yellow_cab.load_data manual__2024-04-30T21:10:20.104666+00:00 [running]> on host 56287055784b
[2024-04-30T21:10:56.075+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_yellow_cab' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-04-30T21:10:20.104666+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-30T21:10:20.104666+00:00'
[2024-04-30T21:10:56.076+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-30T21:10:56.091+0000] {logging_mixin.py:188} INFO - enter for loop
[2024-04-30T21:10:56.091+0000] {logging_mixin.py:188} INFO - enter for loop
[2024-04-30T21:10:56.092+0000] {logging_mixin.py:188} INFO - enter for loop
[2024-04-30T21:10:56.092+0000] {logging_mixin.py:188} INFO - enter for loop
[2024-04-30T21:10:56.093+0000] {logging_mixin.py:188} INFO - enter if loop
[2024-04-30T21:10:56.093+0000] {logging_mixin.py:188} INFO - enter file_upload function
[2024-04-30T21:10:56.093+0000] {logging_mixin.py:188} INFO - read filepath
[2024-04-30T21:10:56.232+0000] {logging_mixin.py:188} INFO - The conn_id `postgresql+psycopg2://***:***@postgres/***` isn't defined
[2024-04-30T21:10:56.235+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-30T21:10:56.236+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-30T21:10:56.244+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_yellow_cab, task_id=load_data, execution_date=20240430T211020, start_date=20240430T211055, end_date=20240430T211056
[2024-04-30T21:10:56.287+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-30T21:10:56.308+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-30T21:10:56.310+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
